### PR TITLE
Azure Network August release

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

Merged ```Network-August-Release``` branch to ```master```

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          "tag1": "value1",
+          "tag2": "value2"
+        },
+        "properties": {
+          "provisioningState": "Succeeded",
+          "resourceGuid": "00000000-0000-0000-0000-000000000000",
+          "virtualNetworkGateway1": {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworkGateways/vpngw"
+          },
+          "localNetworkGateway2": {
```
Comment (jianghaolu) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json]: According to https://github.com/Azure/azure-rest-api-specs/pull/3635/files#diff-795d13db3a317a9440687541d8d00549R2545, `properties` property is required.

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "location": "westus",
+        "tags": {
+          "tag1": "value1",
+          "tag2": "value2"
+        },
+        "properties": {
+          "provisioningState": "Succeeded",
+          "resourceGuid": "00000000-0000-0000-0000-000000000000",
+          "virtualNetworkGateway1": {
+            "id": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/virtualNetworkGateways/vpngw"
```
Comment (jianghaolu) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json]: According to https://github.com/Azure/azure-rest-api-specs/pull/3635/files#diff-795d13db3a317a9440687541d8d00549R2545, `properties` property is required.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json (hunk 1 above)
Comment (MikhailTryakhov) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/VirtualNetworkGatewayConnectionUpdateTags.json]: Guys all these changes were approved in previous PRs. this was just copy-pasted from previous api-version and had no problems as well. every CI job is passed. I see properties above. can't understand a problem

**Comment by MikhailTryakhov** [comment]
@jianghaolu can you please merge it?

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**